packages/automated-dispute/src/services/discordNotifier.ts

```diff
@@ -14,18 +15,23 @@ export type WebhookMessage = WebhookMessageCreateOptions & {
  */
 export class DiscordNotifier implements NotificationService {
     private client: WebhookClient;
+    private defaultAvatarUrl?: string;
+    private logger: Logger;
 
     /**
      * Creates an instance of DiscordNotifier.
      *
      * @param {string} url - The Discord webhook URL.
+     * @param {string} [defaultAvatarUrl] - The default avatar URL to use if none is provided.
      */
-    constructor(url: string) {
+    constructor(url: string, defaultAvatarUrl?: string) {
```

Comment:
Let's inject `logger`. And added some minor tweaks, just being consistent with the rest of our codebase:
```suggestion
    constructor(
      url: string, 
      private readonly logger: ILogger, 
      private readonly defaultAvatarUrl?: string
    ) {
```